portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=IBM+Plex+Sans&display=swap');
+
+:root, body, html {
+  font-size: 16px;
+  font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header, footer {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  max-width: 100vh;
+  width: 80rem;
+  padding: 0 2rem;
+  margin: 0 auto;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#content img {
+  display: inline-block;
```

Comment:
Images default to having `display: inline-block`

Reply:
Removed that rule.